### PR TITLE
Update focus style on buttons

### PR DESCRIPTION
**Pull request by almonais**
## Catégorisation
- :bug: Cette PR est une correction de bug

## Description
Focus on button's
Changes : 
`<div class="button-group>` becomes `<div class="button-group mod-more"` when this button-group has a `<div class="button mod-more">` child

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1037

File: packages/scss/src/components/_button.scss [hunk 1]
```diff
@@ -304,6 +309,20 @@
 .button-group {
 	display: flex;
 
+	&.mod-more {
+		width: fit-content;
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/_button.scss:313]: Ça ne marchera pas avec I.E. 11 ça. :(
Un `display inline-flex` ne serait pas jouable ?

Comment (almonais) [reply packages/scss/src/components/_button.scss:313]: si toutafé ca marche très bien, je le remplace de ce pas !

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1037

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1037

File: packages/scss/src/components/_button.scss [hunk 2]
```diff
@@ -62,6 +62,11 @@
 			background: _color($palette, "light");
 		}
 
+		&:focus {
+			box-shadow: inset 0 0 0 2px _color($palette, "200");
```
Comment (sanlucca) [inline packages/scss/src/components/_button.scss:66]: Looking at the mock-up, it shouldn't be a inset box-shadow. The focus is around the button not inside. 
Also, the mock-up shows a 4px focus ring.

File: packages/scss/src/components/_button.scss [hunk 3]
```diff
@@ -304,6 +309,20 @@
 .button-group {
 	display: flex;
 
+	&.mod-more {
+		display: inline-flex;
+
+		&:focus-within {
+			box-shadow: inset 0 0 0 2px _color("primary","200");
```
Comment (sanlucca) [inline packages/scss/src/components/_button.scss:316]: Same as above

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1037

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1037

**Comment by sanlucca** [comment]
I just realized, there is a shadow on hover and focus states for every type of buttons.

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1037

File: packages/scss/src/components/_button.scss [hunk 4]
```diff
@@ -113,7 +113,7 @@
 			color: _color("primary");
 		}
 
-		@mixin buttonOutlineColoring($palette) {
+		@mixin buttonOutlinkColoring($palette) {
```
Comment (sanlucca) [inline packages/scss/src/components/_button.scss:116]:
```suggestion
		@mixin buttonLinkColoring($palette) {
```

File: packages/scss/src/components/_button.scss [hunk 5]
```diff
@@ -130,12 +130,12 @@
 		}
 
 		// default color
-		@include buttonOutlineColoring("button.default-palette");
+		@include buttonOutlinkColoring("button.default-palette");
```
Comment (sanlucca) [inline packages/scss/src/components/_button.scss:133]:
```suggestion
		@include buttonLinkColoring("button.default-palette");
```

File: packages/scss/src/components/_button.scss [hunk 6]
```diff
@@ -130,12 +130,12 @@
...
 
 		// palette colors
 		@each $name, $palette in _getMap("palettes") {
 			&.palette-#{$name} {
-				@include buttonOutlineColoring($name);
+				@include buttonOutlinkColoring($name);
```
Comment (sanlucca) [inline packages/scss/src/components/_button.scss:138]:
```suggestion
				@include buttonLinkColoring($name);
```

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1037

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1037

**Pull request merged**